src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistorySearch.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import { useSelector, useDispatch } from 'react-redux';
+import { Page, collectionSelectorHelpers, Search } from '@linn-it/linn-form-components-library';
+import bomVerificationHistoryEntriesSearchActions from '../../actions/bomVerificationHistoryEntriesActions';
+import history from '../../history';
+import config from '../../config';
+import { bomVerificationHistoryEntries as bomVerificationHistoryEntriesItemType } from '../../itemTypes';
```

Comment:
probably don't need the 'as bomVerificationHistoryEntriesItemType' alias here, can just import it as is